src/frontend/src/utils/iiConnection.ts

```diff
@@ -345,6 +381,35 @@ export class IIConnection {
     return this.actor;
   }
 
+  lookupAnchorInfo = async (
```

Comment:
shouldn't this be `getAnchorInfo` for consistency with the canister method naming?